man/oxide-msg.1

```diff
@@ -11,7 +25,7 @@ oxide-msg - send messages to oxidewm
 command] [\f[B]-a\f[R] argument]
 .SH DESCRIPTION
 .PP
-The \f[C]oxide-msg\f[R] is an ipc command tool allowing queriying and
+The \f[V]oxide-msg\f[R] is an ipc command tool allowing queriying and
 messaging to oxidewm via the commandline
```

Comment:
To be consistent, please name it "Oxide wm" with a capital O :)

Reply:
I don't know why this file shows up in my pull request, @FelixSchladt wrote the Oxide-msg man page.